src/www/ui/api/Controllers/JobController.php

```diff
@@ -40,6 +41,20 @@
  */
 class JobController extends RestController
 {
+  public function getAllJobs($request, $response, $args)
+  {
+    $query = $request->getQueryParams();
+    $limit = 0;
+    $page = 1;
```

Comment:
Please read the values from the request headers.

Reply:
resolved